content/_data/upgrades/2-302-1.adoc

```diff
@@ -0,0 +1,176 @@
+==== Default Docker images use Java 11
```

Comment:
Is here or somewhere in changelog a good place to mention multi arch docker images being available in this LTS?

Reply:
I was going to include that in the blog post, but this is also a good place to mention it.

Reply:
Included in 0a5bdf1e99be0ebf30347a70754adc4fa107c276